cartographer_grpc/handlers/receive_local_slam_results_handler.h

```diff
@@ -39,7 +39,12 @@ class ReceiveLocalSlamResultsHandler
                 request.trajectory_id(),
                 [writer](std::unique_ptr<MapBuilderServer::LocalSlamResult>
                              local_slam_result) {
-                  writer(GenerateResponse(std::move(local_slam_result)));
+                  if (local_slam_result) {
```

Comment:
Can you add a comment explaining our convention: empty LocalSlamResult => trajectory finished.

Reply:
Done